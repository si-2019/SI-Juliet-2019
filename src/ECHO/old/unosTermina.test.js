import React from "react";
import UnosTermina from "./unosTermina";
import { mount, shallow, render } from "enzyme";

it("Da li se prikazuju komponente", () => {
  const component = shallow(<UnosTermina />);
  expect(component).toMatchSnapshot();
});

it("Da li ima prikaz radnih dana za odabir u formi", () => {
  const strings = ["Ponedjeljak", "Utorak", "Srijeda", "Četvrtak", "Petak"];
  const component = shallow(<UnosTermina list={strings} />);
  expect(component).toMatchSnapshot();
});

it("Da li ima input za unos vremena", () => {
  const wrapper = shallow(<UnosTermina />);
  expect(wrapper.find("#satnica").exists()).toBe(true);
});

it("Da li ima input za unos broja časova", () => {
  const wrapper = shallow(<UnosTermina />);
  expect(wrapper.find("#brCasova").exists()).toBe(true);
});
