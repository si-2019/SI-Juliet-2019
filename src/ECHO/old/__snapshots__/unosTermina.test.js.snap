// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Da li ima prikaz radnih dana za odabir u formi 1`] = `ShallowWrapper {}`;

exports[`Da li se prikazuju komponente 1`] = `ShallowWrapper {}`;
